dbms/src/Storages/DeltaMerge/LocalIndexerScheduler.cpp

```diff
@@ -365,6 +366,8 @@ LocalIndexerScheduler::ScheduleResult LocalIndexerScheduler::scheduleNextTask(st
 
 void LocalIndexerScheduler::schedulerLoop()
 {
+    setThreadName("LocalIndexScheduler-SchedulerLoop");
```

Comment:
This name is too long
https://github.com/pingcap/tiflash/blob/b30c1f5090502d322822cff00131b4abe8bb3631/dbms/src/Common/setThreadName.cpp#L38-L43

Reply:
Fixed